src/doc/tarpl/conversions.md

```diff
@@ -0,0 +1,31 @@
+% Type Conversions
+
+At the end of the day, everything is just a pile of bits somewhere, and type
+systems are just there to help us use those bits right. Needing to reinterpret
+those piles of bits as different types is a common problem and Rust consequently
+gives you several ways to do that.
+
```

Comment:
The end of this para makes me a little bit twitchy - many conversions in Rust do more than just reinterpret the bits, and I worry this might give the wrong idea a little bit.